src/doc/gui_manual/CommandWindow/Vrcfile.rst

```diff
@@ -4,19 +4,15 @@ VisIt Run Commands (RC) File
 ----------------------------
 
 VisIt_ supports a `run commands <https://en.wikipedia.org/wiki/Run_commands>`_
-or *rc* file called the ``visitrc`` file. It is located in the VisIt_ user
-directory, which on most platforms is ``~/.visit``. The ``visitrc`` file
+or *rc* file called the ``visitrc`` file typically
+:ref:`located <file_locations_visitrc>` in ``~/.visit``.  The ``visitrc`` file
 is a Python source code file that contains Python scripting commands that VisIt_
-executes whenever it is started (either via the GUI or the CLI).
+executes whenever the CLI is started either from the shell or from within the
+GUI through the :ref:`**Command Window** <Command_Line_Window>`.
```

Comment:
"**Command Window**" has the asterisks in the html output instead of it being in bold, which I assume is what you intended. The window is called "Commands" not "Command".